index.html

```diff
@@ -26,6 +26,159 @@ <h1 class="career">I'm a software developer</h1>
                 <li><a class="twitter" href="https://twitter.com/Juanse77930"><img src="images/twitter.svg" alt="twitter"></a></li>
             </ul>
         </section>
+        <section>
+            <a class="works"><img src="Backgroud-images/Disabled.png" alt="works-button"></a>
+            <h1 class="recent-works">My Recent Works</h1>
+            <img class="separation-bar" src="Backgroud-images/Indicator.png" alt="separation-bar">
+        </section>
+        <section class="container">
+            <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+            <div class="multi">
+                <section class="multi-post">
+                    <h1 class="multi-post-title">Multi-Post Stories Gain+Glory</h1>
+                    <section class="buttons">
+                        <button class="languages">Ruby on rails</button>
+                        <button class="languages">css</button>
+                        <button class="languages">JavScript</button>
+                        <button class="languages">html</button>
```

Comment:
We should aim to replicate Figma's design as much as possible! As you can see in the following image, we have still some differences in the techs. Kindly make this changes to make your portfolio even better ✔

Figma's
![image](https://user-images.githubusercontent.com/15807343/153528882-a6354672-f8bd-4eee-afaa-9ff5b4ce5c57.png)

Yours 
![Captura](https://user-images.githubusercontent.com/15807343/153528895-cae2c8f3-1cfc-4c06-98cf-4a518ac5ac3c.PNG)